package.json

```diff
@@ -40,7 +40,7 @@
   "scripts": {
     "build-clean": "rm -rf ./packages/*/build",
     "build": "node ./scripts/build.js",
-    "clean-all": "rm -rf ./node_modules; rm -rf ./packages/*/node_modules; rm -rf ./integration_tests/*/*/node_modules; yarn run build-clean",
+    "clean-all": "rm -rf ./node_modules && rm -rf ./packages/*/node_modules && rm -rf ./integration_tests/*/*/node_modules && yarn run build-clean",
```

Comment:
Yeah I started correcting that one too but with the rm command it would blow up anyway. Can't think of a tidy way to do that one without putting a "clean-all-windows" version in? Open to suggestions.